### PR TITLE
Migrate attributes data factory to Angular

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Migrates attributesDataFactory to Angular
Upgrades blueprintFactory and templateEditorFactory, so they can be injected as Angular dependency
Adds type check for lodash, as it is used by attributesDataFactory

At this point I didn't improve the code to follow Typescript annotations, the goal was mainly to have the service running from Angular. 

## Motivation and Context
Angular migration

## How Has This Been Tested?
Locally and on [stage-20]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/app/template-editor/services/attribute-data.service.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import { Injectable } from '@angular/core';
+import { BlueprintFactory, TemplateEditorFactory } from 'src/app/ajs-upgraded-providers';
+import * as angular from 'angular';
+import * as _ from 'lodash';
```
Comment (alex-deaconu) [inline src/app/template-editor/services/attribute-data.service.ts:4]: @ezequielc this obviously works, but I don't see `lodash` being imported through npm. How is this dependency fulfilled, any ideas? Also, should we move `lodash` from bower to npm with this change? Thanks.

**Review by alex-deaconu — APPROVED** [review]
Looks good 👍

File: src/app/template-editor/services/attribute-data.service.ts (hunk 1 above)
Comment (ezequielc) [reply src/app/template-editor/services/attribute-data.service.ts:4]: @alex-deaconu Yeah, I am not sure as well how it is fulfilled. If ok with you, I will merge this PR and try to move `lodash` to npm in a subsequent PR and see how it goes. Thanks

Comment (alex-deaconu) [reply src/app/template-editor/services/attribute-data.service.ts:4]: @ezequielc sounds good, I would not spend too much time on it. Thanks.

Comment (ezequielc) [reply src/app/template-editor/services/attribute-data.service.ts:4]: @alex-deaconu Yeah, agreed.

**Pull request merged**